src/main/java/org/example/Clazz.java

```diff
@@ -0,0 +1,37 @@
+package org.example;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.List;
+
+public class Clazz {
```

Comment:
Suggested change of name to CharacterClass

Reply:
Done